packages/typescript-estree/src/parseSettings/resolveProjectList.ts

```diff
@@ -1,6 +1,6 @@
 import debug from 'debug';
-import { sync as globSync } from 'fast-glob';
 import isGlob from 'is-glob';
+import { globSync } from 'tinyglobby';
```

Comment:
we don't need the `is-glob` package anymore. we can use `isDynamicPattern` instead
```suggestion
import { globSync, isDynamicPattern } from 'tinyglobby';
```